schema.ts

```diff
@@ -1247,6 +1247,14 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    * @hidden
    */
   groupIdProperty?: PropertyIdentifier<K>;
+  /**
+   * The name of a property within {@link ObjectSchemaDefinition.properties} that represents a unique id for a
+   * parent entity for the object. It is recommended for sync table schemas with a bodyTextProperty to specify an
```

Comment:
In theory, could you specify a `bodyTextProperty` without chunking it into multiple rows? If, perhaps clarify that here?